meson.build

```diff
@@ -716,11 +716,20 @@ foreach scheme: get_option('extra-net-naming-schemes').split(',')
                 name = scheme.split('=')[0]
                 value = scheme.split('=')[1]
                 NAME = name.underscorify().to_upper()
-                VALUE = []
+                ENABLED_SCHEMAS = []
+                DISABLED_SCHEMAS = []
                 foreach field: value.split('+')
-                        VALUE += 'NAMING_' + field.underscorify().to_upper()
+                        if field.startswith('-')
+                                DISABLED_SCHEMAS += 'NAMING_' + field.substring(1).underscorify().to_upper()
```

Comment:
`substring()` is only available since 0.56.0. Dunno why meson doesn't complain, though. You should be able to do the same with `strip('-')`.

Reply:
Good catch. Thanks you @medhefgo.